packages/web-components/src/button/button.styles.ts

```diff
@@ -142,10 +142,14 @@ export const baseButtonStyles = css`
     margin-inline-end: var(--icon-spacing);
   }
 
-  :is([slot='end'], ::slotted([slot='end'])) {
+  :host(:not(${iconOnlyState})) :is([slot='end'], :host(:not(${iconOnlyState}))::slotted([slot='end'])) {
```

Comment:
nit: move the second selector to the next line

Reply:
It auto formats this way.